### PR TITLE
cisco: multiasic-qos: Print the suitable message with assert instead of blank assert, for BufferPoolWatermark

### PR DESCRIPTION
**Pull request by rraghav-cisco**
This is a simple change, where the asserts in the BufferPoolWatermark Test are updated to have a assert message instead of a blank.  This helps with easier debugging.

### CODE REVIEW

File: tests/saitests/py3/sai_qos_tests.py
Comment (abdosi) [inline tests/saitests/py3/sai_qos_tests.py:4195]: do we need similar change for other test case also ?

Comment (rraghav-cisco) [reply tests/saitests/py3/sai_qos_tests.py:4195]: Yes, they can also use this change.

Comment (abdosi) [reply tests/saitests/py3/sai_qos_tests.py:4195]: @neethajohn i am surprise why we don;t see this issue more oftern ? When destination port is lag we are not using same packet that calculate lag member port and the one we are using from test case perspective itself.

**Comment by rraghav-cisco** [comment]
@abdosi , Pls merge only after 8377 is merged.

**Review by abdosi — CHANGES_REQUESTED** [review]
We need more understanding before this can be merged.

**Review by XuChen-MSFT — APPROVED** [review]
LGTM

**Comment by rraghav-cisco** [comment]
> @rraghav-cisco Can you help resolve the merge conflicts?

@wangxin : Resolved the conflicts.

**Comment by wangxin** [comment]
Thanks @rraghav-cisco!
@abdosi Can you check if the change looks good to you?

**Comment by XuChen-MSFT** [comment]
@wangxin 
Can you help to merge it?

**Comment by wangxin** [comment]
@abdosi Can you signoff then we can proceed with merge? Currently merging is blocked by your change request.

**Comment by abdosi** [comment]
PR description and subject are not align

**Comment by rraghav-cisco** [comment]
@abdosi : I updated the description.

**Review by abdosi — APPROVED** [review]

**Comment by abdosi** [comment]
@wangxin please help merge this

**Pull request merged**
